### DISCUSSION TITLE
Should NextJS image component load different images with different size?

### DISCUSSION THREAD

**Discussion by qlereboursBS** [vercel/next.js#19327] [Help] [answered]
Hello everyone,

I know that it seems like an issue but I'm not sure, so this is why I think it's better to ask here.
If we look at the demo of the Next Image component with the Intrinsic layout [here](https://image-component.nextjs.gallery/layout-intrinsic), it doesn't seem to change the loaded image (and it's weight) depending on the screen size.
On my laptop, when I'm loading the page in fullsize, it's loading an image with a 18.6kB weight.
If I resize with Chrome tools, to a Phone screen size, the weight is still the same when the page loads.
If we have a look at the srcset, it's only changing the image depending on the resolution.
Is it the expected behavior or is this an issue? I thought it was designed to automatically optimize the loaded image depending on the screen size.

Thanks for your help :)

**Comment by Timer** [comment, marked as answer]
The loaded size only gets automatically adjusted when using the `responsive` or `fill` layouts. Intrinsic images should be sized by you, and are not meant to be "full width' (think avatars at 64x64 pixels).

**Reply by qlereboursBS** [reply]
Thank you very much, it answers my question :+1:
